apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.sql

```diff
@@ -75,3 +75,11 @@ FROM
   ) AS ssq ON ssq.sharing_set_id = ss.id
 WHERE
   ss.course_id = $course_id;
+
+-- BLOCK select_topics_by_course_id
```

Comment:
There's an almost-identical query on the `instructorCourseAdminTopics` page. Maybe we can move a shared query to `models/topic`? That other query includes an `ORDER BY number`, which I think we want here too.

Reply:
Moved this query to `models/topics` and updated to use that both here and in `instructorCourseAdminTopics` in [98f2dd5](https://github.com/PrairieLearn/PrairieLearn/pull/10868/commits/98f2dd558c3f286228530e2a7899b8db990394be).